google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -3276,13 +3525,27 @@ private int numWaiters() {
     }
   }
 
+  @VisibleForTesting
+  int getNumMultiplexedSessionWaiters() {
+    synchronized (lock) {
+      return multiplexedSessionWaiters.size();
+    }
+  }
+
   @VisibleForTesting
   int totalSessions() {
     synchronized (lock) {
       return allSessions.size();
     }
   }
 
+  @VisibleForTesting
+  int totalMultiplexedSessions() {
```

Comment:
Remove this, as it gives the impression that there might be more than 1 multiplexed session